chart/templates/workers/worker-deployment.yaml

```diff
@@ -46,8 +46,13 @@ spec:
       tier: airflow
       component: worker
       release: {{ .Release.Name }}
-  {{ if .Values.workers.updateStrategy }}
-{{ toYaml .Values.workers.updateStrategy | indent 2 }}
+  {{ if and $persistence .Values.workers.updateStrategy }}
+  updateStrategy:
+    {{- toYaml .Values.workers.updateStrategy | nindent 4 }}
+  {{- end }}
+  {{ if and (not $persistence) (.Values.workers.strategy) }}
+  strategy:
+    {{- toYaml .Values.workers.strategy | nindent 4 }}
```

Comment:
@kaxil What's the reason for using `nindent` here?

Reply:
Just want to indent with a new line, without it I would need to have following:

```yaml
  strategy:
{{ toYaml .Values.workers.strategy | nindent 4 }}
```

I prefer (1) as it makes it clear the values are subkeys to strategy: key (just more readable to me)